docs/en/admins/10_ServerConfig.md

```diff
@@ -91,7 +91,9 @@ server {
 		fastcgi_split_path_info ^(.+\.php)(/.*)$;
 		# By default, the variable PATH_INFO is not set under PHP-FPM
 		# But FreshRSS API greader.php need it. If you have a “Bad Request” error, double check this var!
-		fastcgi_param PATH_INFO $fastcgi_path_info;
+		# NOTE: the separate $path_info variable is required (see #3076).
```

Comment:
Sorry, I was not clear enough. I meant the link to NGINX bug.

Reply:
The one you displayed in the issue on our end

Reply:
Done. If you have any issues with the phrasing, please feel free to make modifications on my behalf.

Reply:
LGTM